subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -241,6 +243,12 @@ struct node_rx_ftr {
 #endif /* CONFIG_BT_HCI_MESH_EXT */
 };
 
+/* Meta-information for isochronous PDUs in node_rx_hdr */
+struct node_rx_iso_meta {
+	uint64_t payload_number : 39; /* cisPayloadNumber */
+	uint32_t timestamp;           /* Time of reception */
```

Comment:
There is probably an opportunity here to align these definitions with the ones doen in iso adaptation layer, once this PR is in the bag I can look at following suit. Should the stream be recorded also?

Reply:
I would expect that the stream can be identified through the handle, but I agree that this struct will change depending on how the ISO AL evolves.